src/test/java/org/broadinstitute/hellbender/tools/exome/ACNVModellerUnitTest.java

```diff
@@ -22,17 +26,12 @@
  */
 public final class ACNVModellerUnitTest extends BaseTest {
     private static final String TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/exome/";
-    private static final String FINAL_SEG_FILE_TAG = "sim-final";
 
     private static final String SAMPLE_NAME = "test";
-    private static final File COVERAGES_FILE = new File(TEST_SUB_DIR
-            + "coverages-for-acnv-modeller.tsv");
-    private static final File TUMOR_ALLELIC_COUNTS_FILE = new File(TEST_SUB_DIR
-            + "snps-for-acnv-modeller.tsv");
-    private static final File SEGMENT_FILE =
-            new File(TEST_SUB_DIR + "segments-for-acnv-modeller.seg");  //44 segments (some spurious)
-    private static final File SEGMENTS_TRUTH_FILE = new File(TEST_SUB_DIR
-            + "segments-truth-for-acnv-modeller.seg");  //30 true segments
+    private static final File COVERAGES_FILE = new File(TEST_SUB_DIR + "coverages-for-acnv-modeller.tsv");
```

Comment:
In the next four lines, use the two-argument constructor e.g. `new File(TEST_SUB_DIR, "coverages-for-acnv-modeller.tsv")`